### DISCUSSION TITLE
全面屏机型下充分使用输入法切换键

### DISCUSSION THREAD

**Discussion by MokOopsing** [imfuxiao/Hamster#147] [Ideas]
能否实现单app多键盘实例，达到用输入法切换键（左下小地球）切换中英之类的另一个（属于仓的）键盘

**Comment by imfuxiao** [comment]
您好，您的意思是否想快速切换rime的输入方案？比如有五笔和拼音两个输入方案，你想快速在这两种之间切换。不知道我理解的是否正确。

**Reply by MokOopsing** [reply]
我的意思是，一个app能否提供多个键盘，效果就是，长按小地球时，在出现的键盘列表中可以有多个候选是有一个app提供的，即单应用多键盘实例

**Reply by imfuxiao** [reply]
能否说下这样做的使用场景是什么，也许有更好的方式实现。

**Reply by MokOopsing** [reply]
在ios上即使我们只添加一个键盘，我们依然无法去掉这个地球键，然后任何键盘上目前都是有个“中/英”切换按钮，不如我们为仓做两个键盘实例，一个是“仓输入法-中文”，另一个是“仓输入法-英文”，长按地球键时，这两个会出现在选单中。点击后回来会切换“仓输入法-中文”和“仓输入法-英文”

**Reply by groverlynn** [reply]
Apple應該沒有開放給第三方鍵盤修改地球鍵。Google的GBoard是自己另加了一個齒輪鍵。
![IMG_8965](https://github.com/imfuxiao/Hamster/assets/14979243/cc9ebb17-87aa-48e8-907c-24546f11e527)
